### DISCUSSION TITLE
This feature is obsolete when Next storage feature is enabled. Can we not set storage limit using next storage?

### DISCUSSION THREAD

**Discussion by shubhamdeol** [react-native-async-storage/async-storage#781] [General]
### Proposal

Above lines in doc are confusing. 
Does this means we won't have to set the storage limit as it's unlimited?

I tried using next storage, and it seems that storage limit is still 5mb

**Comment by krizzu** [comment]
With Next Storage, there's no limitation on db size, but rather how big the data can be at reading at one time. That's vary between Android OS, but most common is around 2MB.

**Reply by shubhamdeol** [reply]
I am using redux persist along with async storage. Redux persist loads the data during app startup. Can this be the reason my data is getting lost?
